components/mpas-ocean/cime_config/buildnml

```diff
@@ -1110,6 +1110,8 @@ def buildnml(case, caseroot, compname):
         lines.append('    <var name="vertVelocityTop"/>')
         lines.append('    <var name="zMid"/>')
         lines.append('    <var name="atmosphericPressure"/>')
+        lines.append('    <var name="normalMLEvelocity"/>')
+        lines.append('    <var name="vertMLEVelocityTop"/>')
```

Comment:
This does not have a Registry variable. I think you mean `vertMLEBolusVelocityTop`

Reply:
thanks for the catch, should be fixed